freqtrade/templates/sample_hyperopt.py

```diff
@@ -46,7 +46,7 @@ class SampleHyperOpt(IHyperOpt):
     """
 
     @staticmethod
-    def indicator_space() -> List[Dimension]:
+    def buy_indicator_space() -> List[Dimension]:
```

Comment:
While i think this would be a good change - i think it can't be changed if you'll not want to write compatibility code - as this is now a function that's not known to freqtrade.